asmcomp/closure.ml

```diff
@@ -16,12 +16,14 @@
 (* Introduction of closures, uncurrying, recognition of direct calls *)
 
 open Misc
+open Numbers
 open Asttypes
 open Primitive
 open Lambda
 open Switch
 open Clambda
 
+module VarMap = Ident.Map
```

Comment:
I think this is confusing ("ident"s versus "var"s).  How about just replacing VarMap with Ident.Map throughout?  That seems fine to me, and more clear in fact.